test/unit/controllers/BatchFileUploadUpscanNotificationControllerSpec.scala

```diff
@@ -88,15 +88,15 @@ class BatchFileUploadUpscanNotificationControllerSpec extends PlaySpec with Mock
 
```

Comment:
I suggest importing `ApiSubscriptionFieldsTestData` so that  `ApiSubscriptionFieldsTestData.subscriptionFieldsId` reduces to `subscriptionFieldsId` . I think this would make tests easier to read. This is across all unit tests in this PR